src/main/java/com/condominium/online/condo/controller/DwellerController.java

```diff
@@ -34,19 +34,32 @@ public Dweller saveDweller(@RequestBody(required = true) Dweller dweller) throws
         return this.dwellerService.saveDweller(dweller);
     }
 
-    @RequestMapping(method = RequestMethod.GET)
+    @RequestMapping(method = RequestMethod.GET, value = "/dwellers")
```

Comment:
Should we use the empty URL to find all, like hot/dweller instead of host/dweller/dwellers